### ISSUE TITLE
Define support for repeated fields in env vars

### ISSUE THREAD

**Issue by jcchavezs** [hypertrace/agent-config#32]
Right now [our env vars](https://github.com/hypertrace/agent-config/blob/main/ENV_VARS.md) don't support repeated fields (aka arrays), we should define if we are going to support them and the way to do it. We could settle on only support repeated for scalar types (e.g. string, bool, enums) because representing objects will be very cumbersome.

Some examples in https://github.com/hypertrace/agent-config/blob/main/config.proto#L26 are:

- `HT_PROPAGATION_FORMATS=B3,TRACE_CONTEXT`
- `HT_PROPAGATION_FORMATS=B3|TRACE_CONTEXT`
- `HT_PROPAGATION_FORMATS=B3;TRACE_CONTEXT`

Ping @hypertrace/agent-maintainers

**Comment by mohit-a21** [comment]
I agree that we should only support env vars for scalar types only. `HT_PROPAGATION_FORMATS=B3,TRACE_CONTEXT` For me, I would go for this format.

**Comment by davexroth** [comment]
`,` works for me.

**Comment by jcchavezs** [comment]
any thoughts @pavolloffay ?

**Comment by pavolloffay** [comment]
`HT_PROPAGATION_FORMATS=B3,TRACE_CONTEXT` looks the most natural for me
